docs/install/Fedora.md

```diff
@@ -0,0 +1,23 @@
+#INSTALL
+atomic can be installed through below methods
+
+##YUM/DNF
+atomic RPM is available on Fedora 21 and later. So a simple `yum/dnf install atomic` will install it fine.
+
+##Make
+On Fedora 21, You need to install required build dependencies 
+```
+yum install pylint python-docker-py go-md2man
```

Comment:
ah yup, `yum-builddep atomic` or `dnf builddep atomic` would work better than yum install.

I think all yum commands including `yum-builddep` have been deprecated starting with f22, so might need to replace with `dnf builddep`. yum would still work but will just throw a deprecation warning